package-lock.json

```diff
@@ -7353,7 +7361,8 @@
         "axios": "^0.17.1",
         "bignumber.js": "^6.0.0",
         "typed-promisify": "^0.4.0",
-        "web3-provider-engine": "^13.6.5"
+        "web3-provider-engine": "^13.6.5",
+        "wyvern-js": "github:ProjectOpenSea/wyvern-js#semver:^3.2.1"
       },
```

Comment:
woah how did this get added back? we don't need the `wyvern-js` dependency anymore

Reply:
@alexanderatallah possibly my package-lock.json